lib/pry/pry_class.rb

```diff
@@ -1,7 +1,17 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = proc do
```

Comment:
@r-obert Yes, maybe even deceive Ruby by setting `$HOME` and/or `$XDG_CONFIG_HOME` to somewhere we like.